hasura/migrations/1592547635960_rename_relationship_member_of_meetings_to_guilds_schema_public_table_cr_user/up.yaml

```diff
@@ -0,0 +1,7 @@
+- args:
+    name: member_of_meetings
```

Comment:
Member of meetings to guilds?

Reply:
Yeah, that's an accident. I fixed it in the next migration :/ Do you think we can somehow collapse them into one in PRs?

Reply:
Okay, I can edit it manually, but that's not a great experience.